tests/hci_provider_client/test_hci_pc_markers.py

```diff
@@ -0,0 +1,96 @@
+import logging
```

Comment:
These are all libtest tests. We should move this file or `hci_provider_client` directory to [tests/libtest/](https://github.com/red-hat-storage/ocs-ci/tree/master/tests/libtest).